README.md

```diff
@@ -1,2 +1,46 @@
-# Hello-microverse
-documentation project
+Project Name
+Description the project.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+Live Demo (if available)
+Live Demo Link
```

Comment:
- [ ] `required`
- [ ] here you add the tech stacks you used to make the project 
- [ ] and a live demo if available